tools/RepositoryManagement/DevHome.RepositoryManagement/Views/RepositoryManagementMainPageView.xaml

```diff
@@ -0,0 +1,179 @@
+                ItemsSource="{x:Bind ViewModel.Items, Mode=OneWay}"
+                SelectionMode="None">
+                <ListView.Header>
+                    <Grid
+                        Margin="0,0,0,5"
+                        Padding="0,0,0,5"
+                        BorderBrush="{ThemeResource DividerStrokeColorDefaultBrush}"
+                        BorderThickness="0,0,0,1">
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition />
+                            <ColumnDefinition Width="2*" />
+                            <ColumnDefinition />
+                            <ColumnDefinition />
+                            <ColumnDefinition Width="40" />
+                        </Grid.ColumnDefinitions>
+                        <TextBlock Grid.Column="0" Text="Name" />
+                        <TextBlock Grid.Column="1" Text="Clone path" />
+                        <TextBlock Grid.Column="2" Text="Latest commit" />
+                        <TextBlock Grid.Column="3" Text="Branch" />
+                    </Grid>
+                </ListView.Header>
+                <ListView.ItemTemplate>
+                    <DataTemplate x:DataType="viewmodels:RepositoryManagementItemViewModel">
+                        <Grid ColumnSpacing="10">
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition />
+                                <ColumnDefinition Width="2*" />
+                                <ColumnDefinition />
+                                <ColumnDefinition />
+                                <ColumnDefinition Width="35" />
+                            </Grid.ColumnDefinitions>
+                            <TextBlock
+                                Grid.Column="0"
+                                Text="{x:Bind RepositoryName}"
+                                ToolTipService.ToolTip="{x:Bind RepositoryName}" />
+                            <TextBlock
+                                Grid.Column="1"
+                                Text="{x:Bind ClonePath}"
+                                ToolTipService.ToolTip="{x:Bind ClonePath}" />
+                            <TextBlock
+                                Grid.Column="2"
+                                Text="{x:Bind LatestCommit}"
+                                ToolTipService.ToolTip="{x:Bind LatestCommit}" />
+                            <TextBlock
+                                Grid.Column="3"
+                                Text="{x:Bind Branch}"
+                                ToolTipService.ToolTip="{x:Bind Branch}" />
+                            <Button
+                                x:Uid="MoreOptionsButton"
+                                Grid.Column="4"
+                                Style="{StaticResource HorizontalThreeDotsStyle}">
+                                <Button.Flyout>
+                                    <MenuFlyout>
```

Comment:
Why are these defined twice?

Reply:
To get the columns to align with the columns.  I did the same with RepoConfig.

The heading of the ListView is a grid and the columns of the header grid does not align with the columns in the list view item.  They are defined twice because the custom control needed to get all columns to align.